### ISSUE TITLE
Every thumbnail in the downloads page is loaded at full resolution, resulting in a long load time

### ISSUE THREAD

**Issue by RaineGit** [Geoxor/geoxor-website#1]
## Issue
As the title says, the thumbnails are loaded as high resolution images, some even get to 6000x6000 pixels.
## Issue (more in depth)
Example of one of the URLs used for linking to one of the thumbnail images:
https://cdn.discordapp.com/attachments/553776591910600714/776821021872750592/Faerie_Cover.png?width=168&height=168
At the end of the URL there are parameters that specify the size at which the image should be loaded, but they are ignored.
## Solution
The solution is to use the `media.discordapp.net` domain instead of `cdn.discordapp.com`

## Before
https://cdn.discordapp.com/attachments/553776591910600714/776821021872750592/Faerie_Cover.png?width=168&height=168
<img src="https://cdn.discordapp.com/attachments/553776591910600714/776821021872750592/Faerie_Cover.png?width=168&height=168"></img>
(you may not be able to see the image because it exceeds GitHub's size limit)
## After
https://media.discordapp.net/attachments/553776591910600714/776821021872750592/Faerie_Cover.png?width=168&height=168
<img src="https://media.discordapp.net/attachments/553776591910600714/776821021872750592/Faerie_Cover.png?width=168&height=168"></img>

**Comment by Geoxor** [comment]
yeah im aware about that, i will remake the entire website soon cus its pretty borked and make it work correctly there
